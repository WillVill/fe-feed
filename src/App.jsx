import React from 'react';
import { PagesRoutes } from './pages';

const App = () => (
	<PagesRoutes />
);

export default App;
