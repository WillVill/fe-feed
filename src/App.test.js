import React from 'react';
import App from './App';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

// Test Setup
const setup = propOverrides => {
	const wrapper = shallow(<App />);

	return {
		wrapper
	};
};

describe('[Component] WorkplaceSelection', () => {
	it('renders', () => {
		const { wrapper } = setup();

		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
