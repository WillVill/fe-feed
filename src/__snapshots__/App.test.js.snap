// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] WorkplaceSelection renders 1`] = `<Component />`;
